_gsocproposals/2022/proposal_CERNBox-Etherpad.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Etherpad plugin as a ScienceMeshDocs editor
+layout: gsoc_proposal
+project: CERNBox
+year: 2022
+organization:
```

Comment:
Project difficulty, duration and mentor availability required:
    difficulty: [medium or high]
    duration: [350 or 175 hours]
    mentor_avail: [month interval]